### PR TITLE
Fix package control having incorrect size and shasum

### PR DESCRIPTION
**Pull request by Newbytee**
While I'm aware of that this might not be the ideal solution in the long term, I think having the package work while a better solution is figured out is a good idea.

Fixes https://github.com/flathub/com.sublimetext.three/issues/20

### CODE REVIEW

**Comment by LeandroStanger** [comment]
@ptomato

**Comment by ptomato** [comment]
@LeandroStanger Please don't send me pings on a day-old pull request. I already get notifications of this pull request and will get to it when I can. I am maintaining this in my free time, this was a holiday weekend for me, and in general I am just not going to have a chance to get to every pull request within 24 hours.

(For future reference, in general I've heard it's polite to send a reminder after about 4 to 7 days.)

File: com.sublimetext.three.yaml [hunk 1]
```diff
@@ -83,8 +83,8 @@ modules:
       - type: extra-data
         filename: Package Control.sublime-package
         url: http://packagecontrol.io/Package%20Control.sublime-package
-        sha256: 6f4c264a24d933ce70df5dedcf1dcaeeebe013ee18cced0ef93d5f746d80ef60
-        size: 286331
+        sha256: 4e0764963c9abc0fe3c1f8fbb70f9c8cf3a14cf3f9ef62bf5175233b02008499
```
Comment (ptomato) [inline com.sublimetext.three.yaml:86]: This is a different size and checksum than I get when I download the file myself: I have `817937144c34c84c88cd43b85318b2656f9c3fac02f8f72cbc18360b2c26d139` and 471460. Could it be that they have changed the file again in the intervening time?

**Review by ptomato — COMMENTED** [review]
Thanks for the pull request! Unfortunately it seems they have changed the file again and that suggests they might do so in the future as well?

File: com.sublimetext.three.yaml (hunk 1 above)
Comment (Newbytee) [reply com.sublimetext.three.yaml:86]: > Could it be that they have changed the file again in the intervening time?

Fortunately I did not delete the file when I downloaded it to get size and checksum and yes, I can confirm that the one I download now is a different file from the one I downloaded previously. Seems they changed it again. I'll push the new shasum and size in a moment.

**Comment by Newbytee** [comment]
Bot, build

**Review by ptomato — APPROVED** [review]
Thanks. This seems to work, at least until they release the next version. I have opened https://github.com/wbond/package_control/issues/1508 to discuss the problem with Package Control.

**Pull request merged**